neo/Consensus/ConsensusService.cs

```diff
@@ -281,9 +286,10 @@ private void OnRecoveryMessageReceived(ConsensusPayload payload, RecoveryMessage
         {
             if (message.ViewNumber < context.ViewNumber) return;
             Log($"{nameof(OnRecoveryMessageReceived)}: height={payload.BlockIndex} view={message.ViewNumber} index={payload.ValidatorIndex}");
+            if (context.WatchOnly()) return;
```

Comment:
Recovery payload may also move nodes view.

Reply:
`WatchOnly` nodes won't request a `Recovery` message.

Reply:
Exactly, but they will receive and process changeviews, prepreq and commits.

PrepResponse are not needed.